### PR TITLE
Documentation Baseline

### PR DESCRIPTION
**Pull request by ghost**
## Description of Changes:

Updated documentation to build with Sphinx and publish to [readthedocs.org](https://garnet-neutrons.readthedocs.io/en/latest/)

## To Test:

```bash
cd /path/to/my/local/garnet/repo/
git fetch origin pull/<PULL_REQUEST_NUMBER>/head:pr<PULL_REQUEST_NUMBER>
git switch pr<PULL_REQUEST_NUMBER>
conda activate <garnet_environment>
python -m pytest <item_to_test>
```

## Check list for the pull request
- [ ] I have read the [CONTRIBUTING]
- [ ] I have read the [CODE_OF_CONDUCT]
- [ ] I have added tests for my changes
- [ ] I have updated the documentation accordingly

---

## Check list for the reviewer
- [ ] I have read the [CONTRIBUTING]
- [ ] I have verified the proposed changes
- [ ] best software practices
    + [ ] all internal functions have an underbar, as is python standard
    + [ ] clearly named variables (better to be verbose in variable names)
    + [ ] code comments explaining the intent of code blocks
- [ ] All the tests are passing
- [ ] The documentation is up to date
- [ ] code comments added when explaining intent

### CODE REVIEW

File: src/garnet/home/view.py [hunk 1]
```diff
@@ -5,7 +5,11 @@
 
 
 class HomeView(QWidget):
-    """Home widget"""
+    """Home widget
+
+    :param QWidget: optional, the main widget
```
Comment (mpatrou) [inline src/garnet/home/view.py]: "optional" can be omitted

File: docs/source/index.rst [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+The garnet tool will allow users to select single crystal diffraction data from one (minimally white beam) or more (minimally monochromatic beam) orientations, and transform it into a meaningful form.
+There exists essential steps of a single crystal data reduction.
+These include:
+- UB matrix determination and refinement for data reduction and experiment planning
+- Peak integration and corrections for structure refinement
+- Reciprocal space reconstruction for visualization and analysis
+- Order parameter tracking and event filtering analysis
+
+Data processing will be based on `Mantid <https://www.mantidproject.org/>`_ and use `PyQt5 <https://pypi.org/project/PyQt5>`_ for the application.
```
Comment (mpatrou) [inline docs/source/index.rst]: we follow Mantid's PyQt version. no need to refer to a specific version here.

Comment (ghost) [reply docs/source/index.rst]: This is a copy/paste of [Garnet Project README](https://github.com/neutrons/garnet/blob/d071108e7a405cb6cce99fbf62296f66fb4e1c29/README.md#L43). Would you like me to update the README as well?

Comment (mpatrou) [reply docs/source/index.rst]: yes, that would be nice. Let's avoid talking about versions right now. :)

**Review by mpatrou — APPROVED** [review]
works as expected.

**Pull request merged**